net/initiate.go

```diff
@@ -87,9 +88,11 @@ func (h *Host) handleProtocolStream(stream libp2pnetwork.Stream) {
 		return
 	}
 
+	curPeer := stream.Conn().RemotePeer()
+
 	log.Debug(
 		"received message from peer, peer=",
-		stream.Conn().RemotePeer(),
+		curPeer,
 		" type=",
 		message.TypeToString(msg.Type()),
 	)
```

Comment:
change this to a `Debugf`? formatting is kinda weird rn lol

Reply:
Done, new message is:
```
log.Debugf("received message from peer=%s type=%s", curPeer, message.TypeToString(msg.Type()))
```